### PR TITLE
US251 Rename solution to HCA

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/Foundation/Account/code/Constants.cs [hunk 1]
```diff
@@ -12,15 +12,15 @@
...
 {
     public static class Constants
     {
         public static class Pipelines
         {
-            public const string Login = "wooli.foundation.account.login";
+            public const string Login = "HCA.foundation.account.login";
 
-            public const string Logout = "wooli.foundation.account.logout";
+            public const string Logout = "HCA.foundation.account.logout";
```
Comment (yuya-yuliya) [inline src/Foundation/Account/code/Constants.cs:23]: Maybe there better to use lowercase?

File: src/Foundation/Commerce/code/App_Config/Include/Foundation/HCA/Foundation.Commerce.config [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          </sites>
+        </processor>
+      </httpRequestBegin>
+      <commerce.orders.submitVisitorOrder>
+        <processor
+          type="HCA.Foundation.Commerce.Infrastructure.Pipelines.SubmitVisitorOrder.SendEmailProcessor, HCA.Foundation.Commerce"
+          resolve="true"
+          patch:after="processor[@type='Sitecore.Commerce.Pipelines.Orders.TriggerOrderOutcome, Sitecore.Commerce.Connect.Core']" />
+      </commerce.orders.submitVisitorOrder>
+      <HCA.foundation.account.login>
```
Comment (yuya-yuliya) [inline src/Foundation/Commerce/code/App_Config/Include/Foundation/HCA/Foundation.Commerce.config:33]: Maybe there better to use lowercase `hca`?

File: src/Project/HCA/code/HCA.Project.Wooli.csproj.DotSettings [hunk 3]
```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=app_005Fconfig/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (yuya-yuliya) [inline src/Project/HCA/code/HCA.Project.Wooli.csproj.DotSettings:2]: Remove this file

File: src/Project/HCA/code/HCAoject.Wooli.csproj.DotSettings [hunk 4]
```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/CodeInspection/NamespaceProvider/NamespaceFoldersToSkip/=app_005Fconfig/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```
Comment (yuya-yuliya) [inline src/Project/HCA/code/HCAoject.Wooli.csproj.DotSettings:2]: Remove this file

File: src/Project/HCA/serialization.content/Content/HCA/Home.yml [hunk 5]
```diff
@@ -57,7 +57,7 @@ Languages:
               uid="{7C92BA41-44DD-47AD-A232-65143CBBF3F4}"
               p:before="*"
               s:key="undefined"
-              s:md="/sitecore/layout/Placeholder Settings/Wooli/Project/Wooli/wooli-content" />
+              s:md="/sitecore/layout/Placeholder Settings/HCA/Project/HCA/wooli-content" />
```
Comment (yuya-yuliya) [inline src/Project/HCA/serialization.content/Content/HCA/Home.yml:60]: Did you rename placeholder setting `wooli-content`?

**Review by yuya-yuliya — COMMENTED** [review]
- Remove `src/Project/HCA/code/Wooli.Project.Wooli.csproj.DotSettings` file
I have some questions:
- as we don't rename our site to HCA in Sitecore, did you rename Storefront settings root and site root?
- in some projects we have .DotSettings file. Let's delete this files

File: src/Project/HCA/serialization.content/Content/HCA/Home.yml (hunk 5 above)
Comment (daria-larionova) [reply src/Project/HCA/serialization.content/Content/HCA/Home.yml:60]: yes, later I'll change this one and host name

Comment (larionovadaria) [reply src/Project/HCA/serialization.content/Content/HCA/Home.yml:60]: yes, later I'll change this one and host name

**Pull request merged**